README.md

```diff
@@ -66,7 +66,13 @@ a number of programs to be installed:
 
 -   [Template Toolkit].  The Debian package is `libtemplate-perl`
 -   [Pandoc].  The Debian package is `pandoc`
--   git
+-   `make`, `wget`, `git`, `perl`, `python3`, `gcc`, `libssl-dev`, and `libz-dev`
```

Comment:
gcc?  libssl-dev?  libz-dev?  We're not compiling anything here, so why?

Reply:
libssl and libz were needed for either NET:: or IO:: module. One of the Modules wanted these libs and in turn gcc... 
I didn't keep build logs, unfortunately. This is what I had to install to get out from the makefile with 0.

These are probably the deps to build OpenSSL::Query deps.

Reply:
> These are probably the deps to build OpenSSL::Query deps.

I suspected this much, and that's why they don't really belong here.